.github/workflows/test-suite.yml

```diff
@@ -231,21 +291,28 @@ jobs:
       run: cargo run --release --bin simulator syncing-sim
   doppelganger-protection-test:
     name: doppelganger-protection-test
-    runs-on: ubuntu-latest
-    needs: cargo-fmt
+    runs-on: ${{ github.repository == 'sigp/lighthouse' && fromJson('["self-hosted", "linux", "CI", "small"]') || 'ubuntu-latest'  }}
     steps:
     - uses: actions/checkout@v3
     - name: Get latest version of stable Rust
-      run: rustup update stable
+      if: env.SELF_HOSTED_RUNNERS == false
+      uses: moonrepo/setup-rust@v1
+      with:
+          channel: stable
+          cache-target: release
+          components: rustfmt
     - name: Install geth
+      if: env.SELF_HOSTED_RUNNERS == false
       run: |
           sudo add-apt-repository -y ppa:ethereum/ethereum
           sudo apt-get update
           sudo apt-get install ethereum
-    - name: Install lighthouse and lcli
+    - name: Install lighthouse
       run: |
           make
-          make install-lcli
+    - name: Install lcli
+      if: env.SELF_HOSTED_RUNNERS == false
+      run: make install-lcli
```

Comment:
We lose the check for compatibility between testnet & `lcli` here (that could potentially catch `lcli` breakage), so we might get feedback for this a week late - but I'm sure this has been considered, and I don't mind it if we can get good gains from this.

Reply:
There's also a chance that this test would keep failing  if there's changes that make `lcli` incompatible with the latest BN, and we'd have to update the runner image to the get test to pass.

Reply:
Yep, we have considered this. We can add it back, but we figured we'd leave it out until it causes pain. Its not often we have these incompatibilities, i think (famous last words)

Reply:
Sounds good leaving it until it causes pain 👍